plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt

```diff
@@ -13,8 +13,52 @@ IV.   KNOWN ISSUES
 
 = I. MAIN CHANGES =
 
-Main changes since Platform 4.4.0
+Main changes since Platform 4.4
 
+* Features
+** Enterprise Skin
+** Web Conferencing
+** Like on comments
+** Insert images in microblog
+** Share multiple documents in microblog Stream
+** Space and profile banner
+** Improved navigation
+** Reply to comment
+** Combine search terms and tags
+** Improved Chat app communication layer thanks to WebSockets
+* Improvements
+** Pluggable USer Profile IM list
+** Configurable spaces administrators
+** Commons RDBMS
+** New Cache strategy & configuration
+** New Organization Service Cache
+** Better Files indexing and search (in Elasticsearch)
+** Improved custom skin mechanisms
+** Hot JCR workspace reindexing
+** New MBean to reload Javascripts
+** Javascript modules minification disabling option
+** Java 9 compatibility (runtime)
+* Upgrades
+** Upgrade to Tomcat 8.5
```

Comment:
We may have to mention that do not support CAS 4 anymore and that we upgraded to CAS 5.1